Updated the templates with changes for JRS 7.5

astoykovtibco opened the pull request:
Description of changes:
- Some labels changed to bette describe the Input Fields 
- MaxLength for the endpoint changes to 128 
- Introduced "ksp" ssh variable to find the key-store 


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

sshvans commented on templates/tibco-jrs-cluster-existing-infrastructure.template:
```diff
@@ -1,12 +1,12 @@
 {
```
I am not sure why the description has been changed along with labels? The earlier version was in compliance with our guidelines. Therefore I would suggest to keep it as is and only make changes which are required to fix the failing test issue.

astoykovtibco commented:
Labels are now reverted. 
Only this two change left:

- MaxLength for the endpoint changes to 128
- Introduced "ksp" ssh variable to find the key-store
